src/address-watch/agent.spec.js

```diff
@@ -6,28 +6,35 @@ const {
   FindingSeverity,
 } = require('forta-agent');
 
-// local definitions
-const { handleTransaction } = require('./agent');
-
-// load config file
-const config = require('../agent-config.json');
-
-// load configuration data from agent config file
 const {
-  developerAbbreviation: developerAbbrev,
-  protocolName,
-  protocolAbbrev,
-  contracts,
-} = config;
+  initialize,
+  handleTransaction
+} = require('./agent');
 
-// make sure addresses is populated
-const addresses = Object.values(contracts);
-if (addresses.length === 0) {
-  throw new Error('Must supply at least one address to watch');
-}
+const config = {
+  developerAbbreviation: "DEVTEST",
+  protocolName: "PROTOTEST",
+  protocolAbbreviation: "PT",
+  agentType: "admin-events",
+  name: "test-agent",
+  contracts: {
+    contractName1: {
+      name: "accountName1",
+      address: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+      watch : {
+        type: "Info",
+        severity: "Info"
+      }
+    }
```

Comment:
This looks different than the example configuration file.  Does the example file need to be updated?